lib/wallaby/driver.ex

```diff
@@ -83,6 +83,14 @@ defmodule Wallaby.Driver do
   end
 
   @doc """
+  Gets the title of the current page.
+  """
+  def title(session) do
+    resp = request(:get, "#{session.base_url}session/#{session.id}/title")
```

Comment:
Can this (is this) function specifically target `<head><title>Text</title></head>` ?  

It's possible that a `<title>` tag can be placed anywhere in a HTML document (obviously not a clever thing for a HTML developer to do, but it is possible), but we are only interested in the title that the browser uses.